src/main/resources/application.yaml

```diff
@@ -168,4 +168,10 @@ acas:
   api:
     url: ${ACAS_BASE_URL:https://api-dev-acas-01.azure-api.net/ECCLUAT}
     key: ${ACAS_KEY:AAAAAAAAAAAAAAAAA}
-    # This environment variable can be set in your local environment. Do not change this value.
\ No newline at end of file
+    # This environment variable can be set in your local environment. Do not change this value.
+hmcts_service_id: BHA1
+
+launchdarkly:
+  sdk-key: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
+  offline-mode: false
+  env: ${LAUNCH_DARKLY_ENV:default}
```

Comment:
Where is this being overridden? Does the default just default to the test environment?